src/utils/events/move-input.js

```diff
@@ -0,0 +1,32 @@
+const MOVE_EVENTS = ['pointermove', 'touchmove', 'mousemove'];
+
+/**
+ * Hammer.js swallows 'move' events (for pointer/touch/mouse)
+ * when the pointer is not down. This class sets up a handler
+ * specifically for these events to work around this limitation.
+ * Note that this could be extended to more intelligently handle
+ * move events across input types, e.g. storing multiple simultaneous
+ * pointer/touch events, calculating speed/direction, etc.
+ */
+export default class MoveInput {
```

Comment:
Nit: This class really doesn't do much. It seems to be there mostly to match `WheelInput`. I wonder if they should just be combined? Or is there an intention to add more of these?